tools/run_tests/artifacts/distribtest_targets.py

```diff
@@ -395,7 +395,6 @@ def targets():
                        testcase='cmake_as_externalproject',
                        presubmit=True),
         # C#
-        CSharpDistribTest('linux', 'x64', 'jessie', presubmit=True),
         CSharpDistribTest('linux', 'x64', 'stretch'),
```

Comment:
add `presubmit=True` to `CSharpDistribTest('linux', 'x64', 'stretch')` (since you're removing jessie that had `presubmit=True`)

Reply:
Done!